lib/schedule.js

```diff
@@ -348,9 +348,9 @@ function recurMatch(val, matcher) {
 
   if (typeof matcher === 'number' || typeof matcher === 'string') {
     return (val === matcher);
-  } else if (typeof matcher === 'object' && matcher instanceof Range) {
+  } else if (matcher instanceof Range) {
     return matcher.contains(val);
-  } else if (Array.isArray(matcher) || (typeof matcher === 'object' && matcher instanceof Array)) {
+  } else if (Array.isArray(matcher) || (matcher instanceof Array)) {
```

Comment:
@santigimeno I've seen this way of checking for arrays before. Some quick search shows that Array.isArray is new as ES5, so I'm guessing it's a fall back for older browsers. Not a problem for us since this is not running in a browser without browserify or webpack. At some point I'll go through these checks since we could probably do a lot of clean up.